src/status_im/contexts/onboarding/events.cljs

```diff
@@ -107,28 +108,15 @@
               [:navigate-to-within-stack [:screen/onboarding.enable-biometrics from-screen]]
               [:onboarding/create-account-and-login])]]})))
 
-(rf/reg-event-fx
- :onboarding/multiaccount-already-exists
- (fn [_ [key-uid]]
-   {:fx [[:effects.utils/show-confirmation
-          {:title               (i18n/label :t/multiaccount-exists-title)
-           :content             (i18n/label :t/multiaccount-exists-content)
-           :confirm-button-text (i18n/label :t/unlock)
-           :on-accept           (fn []
-                                  (re-frame/dispatch [:pop-to-root :screen/profile.profiles])
-                                  ;; Note - Profile selection is not working
-                                  (re-frame/dispatch [:profile/profile-selected key-uid]))
-           :on-cancel           #(re-frame/dispatch [:pop-to-root :screen/profile.profiles])}]]}))
-
 (rf/reg-event-fx
  :onboarding/seed-phrase-validated
- (fn [{:keys [db]} [seed-phrase key-uid]]
+ (fn [{:keys [db]} [{:keys [seed-phrase key-uid]}]]
    (let [next-screen :screen/onboarding.create-profile-password
          from-screen (get db
                           :onboarding/navigated-to-enter-seed-phrase-from-screen
                           :screen/onboarding.create-profile)]
      (if (contains? (:profile/profiles-overview db) key-uid)
-       {:fx [[:dispatch [:onboarding/multiaccount-already-exists key-uid]]]}
+       {:fx [[:dispatch [:enter-seed-phrase/set-error (i18n/label :t/account-already-exist-error)]]]}
```

Comment:
i believe here we should keep multiaccount-already-exists because we want user be able to login

Reply:
Hi, user will be still able to login by going back manually, we are just removing popup.

Original discussion - https://github.com/status-im/status-mobile/pull/21947#pullrequestreview-2563056380

> I don't have any strong preference regarding this point, but I find the current solution showing a pop-up slightly worse than what's in Figma because the user loses context by being redirected to the profile selection. The solution in Figma is consistent with other error messages that can occur in the recovery phrase screen, so that's mainly why I prefer it. If the effort is small seems worth improving to match the UX in Figma. Not a blocker to this PR IMO.